CMakeLists.txt

```diff
@@ -7,12 +7,13 @@
 #   target_link_libraries(your_target mshio::mshio)
 #
 # ============================================================================
-
+message(STATUS "CMake version: ${CMAKE_VERSION}")
 cmake_minimum_required(VERSION 3.11)
 set(CMAKE_CXX_STANDARD 14)
 
 project(MshIO)
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Thanks! I recommend setting this per-target instead of modifying a global cmake flag:

```cmake
set_property(TARGET lib1 PROPERTY POSITION_INDEPENDENT_CODE ON)
```